trackintel/analysis/location_identification.py

```diff
@@ -197,6 +197,8 @@ def freq_method(staypoints, *labels):
             group["duration"] = group["finished_at"] - group["started_at"]
```

Comment:
It's not related to this PR, but I guess this line can be moved out of the `for name, group in sp.groupby("user_id"):` for better readability?

Reply:
I think I did it so I wouldn't have to delete it from the `sp` `DataFrame` later, but I'm not sure.

Reply:
good point, I'll resolve this comment